main.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
You just have 1 package at the moment, `main`. This is fine for now, but as the project grows you'll probably want to organise stuff a bit better. Once you do that, you'll need to consider the names of your functions/structs/fields, as they're all private (start with a lower case letter) at the moment. Once you have more packages, you'll need to think about the interfaces between your packagers, and what things should be accessible outside or not